packages/core/backend/sirius-components-collaborative/src/main/java/org/eclipse/sirius/components/collaborative/dto/ProjectEventInput.java

```diff
@@ -0,0 +1,25 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.collaborative.dto;
+
+import java.util.UUID;
+
+import org.eclipse.sirius.components.core.api.IInput;
+
+/**
+ * The input of the project event subscription.
+ *
+ * @author arichard
+ */
+public record ProjectEventInput(UUID id, UUID projectId) implements IInput {
```

Comment:
To be moved in Sirius Web, Sirius Components does not have any notion of project